docs/project/find-an-issue.md

```diff
@@ -61,79 +55,76 @@ for you to know that triage includes ranking issues according to difficulty.
+      <td class="tg-031e"><strong class="gh-label expert">exp/expert</strong></td>
+      <td class="tg-031e">You have made less than 20 commits to Docker which amount to 500-1000 code lines or 1000-3000 documentation lines. </td>
+    </tr>
+    <tr>
+      <td class="tg-031e"><strong class="gh-label master">exp/master</strong></td>
+      <td class="tg-031e">You have made more than 20 commits to Docker and greater than 1000 code lines or 3000 documentation lines.</td>
+    </tr>
+  </tbody>
 </table>
 
-As the table states, these labels are meant as guidelines. You might have
-written a whole plugin for Docker in a personal project and never contributed to
-Docker. With that kind of experience, you could take on an <strong
+These labels are guidelines. You might have written a whole plugin for Docker in a personal 
+project and never contributed to Docker. With that kind of experience, you could take on an <strong
 class="gh-label expert">exp/expert</strong> or <strong class="gh-label
-master">exp/master</strong> level task.
+master">exp/master</strong> level issue.
 
 ## Claim a beginner or novice issue
 
-In this section, you find and claim an open documentation lines issue.
-
+To claim an issue:
 
 1. Go to the `docker/docker` <a
 	href="https://github.com/docker/docker" target="_blank">repository</a>.
 
-2. Click on the "Issues" link.
+2. Click the "Issues" link.
 
     A list of the open issues appears. 
 
     ![Open issues](/project/images/issue_list.png)
 
-3. Look for the <strong class="gh-label beginner">exp/beginner</strong> items on the list.
-
-4. Click on the "labels" dropdown and select  <strong class="gh-label beginner">exp/beginner</strong>.
+3. From the "Labels" drop-down, select <strong class="gh-label beginner">exp/beginner</strong>.
 
     The system filters to show only open <strong class="gh-label beginner">exp/beginner</strong> issues.
 
-5. Open an issue that interests you.
+4. Open an issue that interests you.
 
-    The comments on the issues can tell you both the problem and the potential 
+    The comments on the issues describe the problem and can provide information for a potential 
     solution.
 
-6. Make sure that no other user has chosen to work on the issue.
+5. When you find an open issue that both interests you and is unclaimed, add a
+`#dibs` comment. Make sure that no other user has chosen to work on the issue.
 
-    We don't allow external contributors to assign issues to themselves. So, you
-    need to read the comments to find if a user claimed the issue by leaving a
-    `#dibs` comment on the issue.  
+    The project does not permit external contributors to assign issues to themselves. Read 
+    the comments to find if a user claimed the issue by leaving a
+    `#dibs` comment on the issue.
 
-7. When you find an open issue that both interests you and is unclaimed, add a
-`#dibs` comment.
+7. Your issue # will be different depending on what you claimed. After a moment, Gordon the Docker 
```

Comment:
Oh, looks like you skipped 6

Since you want to contribute more;
Sometimes reviewers leave a comment like `s/foo/bar/`. You may not be familiar with that; this is shorthand for "replace 'foo' with 'bar'". It's basically how one would replace a text with `sed`, and it's to save time reviewing.